### PR TITLE
Update UG with archive and saving data features

### LINKED ISSUE #3: Update the User Guide

### PR DESCRIPTION
**Pull request by yanbingtao**
Update UG (#3) by adding documentation for archive and saving data features

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -150,9 +150,32 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+All tCheck data (i.e. contact details, ingredient data, sales data) are saved in the hard disk automatically after any
+ command that changes the data. There is no need to save manually.
 
-### Archiving data files `[coming in v2.0]`
+### Archiving employees' contact details
```
Comment (ureshiiYing) [inline docs/UserGuide.md:139]: Perhaps this archive employees contact details section should be grouped with the other employee contact details features?

**Review by ureshiiYing — COMMENTED** [review]
Overall, LGTM. Perhaps add headers to the employee contact details section since it is only one section of tCheck features?

File: docs/UserGuide.md (hunk 1 above)
Comment (yanbingtao) [reply docs/UserGuide.md:139]: Good point! Resolved, thanks (:

**Comment by WM71811** [comment]
LGTM

**Pull request merged**